example/newreposecret/main.go

```diff
@@ -0,0 +1,197 @@
+package main
```

Comment:
To follow idiomatic Go style and to work nicely with `godoc`, please put the package comment here. For example:

```
// newreposecret creates a new secret in GitHub for a given ...
// 
// Usage:
// ...
//
// Example:
// ...
package main
```

Please write this with the Godocs in mind... use complete sentences ending in periods. Capitalize new sentences, capitalize GitHub correctly, etc.

Then you can completely delete lines 21-37 below.

Reply:
updated!

Reply:
I went through and updated all the places you mentioned to update and pushed a new commit. Thank you so much for taking the time to review my code so thoroughly and precisely. I really appreciate you taking the time to clean this up with me. I'm no pro gopher so I really appreciate everything you are doing to work with me! I wasn't sure if I should resolve the conversations or you, so I left them open so that you could resolve them if my updates met your expectations. Thank you again for all your work on this! I'm really excited to potentially get this merged some time 😃